src/client/components/map/ProjectDistrictsMap.tsx

```diff
@@ -0,0 +1,110 @@
+/** @jsx jsx */
+import { Box, jsx, Spinner } from "theme-ui";
+import MapboxGL from "mapbox-gl";
+import React, { useEffect, useRef, useState } from "react";
+import bbox from "@turf/bbox";
+
+import { BBox2d } from "@turf/helpers/lib/geojson";
+
+import { ProjectNest } from "../../../shared/entities";
+import { DistrictGeoJSON } from "../../types";
+import { getDistrictColor } from "../../constants/colors";
+
+const style = {
+  mapContainer: {
+    height: "250px",
+    display: "inline-block",
+    position: "relative",
+    p: "15px"
```

Comment:
I assume you had this at some point and lost it?
All of the flyout menus were hiding up in the upper-right corner of the screen w/o this.
```suggestion
    p: "15px",
    position: "relative"
```